lib/DueDatePicker/index.js

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DayPicker from 'react-day-picker';
+import Overlay from 'react-bootstrap/lib/Overlay';
+import Popover from 'react-bootstrap/lib/Popover';
+import moment from 'moment';
+import cx from 'classnames';
+import Button from '../Button';
+import DueDateHeader from './DueDateHeader';
+import DueDateButton from './DueDateButton';
+
+class DueDatePicker extends Component {
+  constructor(props) {
+    super(props);
+    this.handleDayClick = this.handleDayClick.bind(this);
+    this.handleTimeClick = this.handleTimeClick.bind(this);
+    this.applyDueDate = this.applyDueDate.bind(this);
+    this.removeDueDate = this.removeDueDate.bind(this);
+
+    this.initialState = {
+      showing: false,
+      changed: false,
+      selectedDay: this.props.dueDate ? this.props.dueDate.toDate() : null,
```

Comment:
I didn't think you had access to `this.props` in the constructor. That's why `props` is passed through as an argument. I'd test this a little further and use the argument.